docs/UserGuide.md

```diff
@@ -275,9 +321,9 @@ OneShelf data are saved in the hard disk automatically after any command that ch
 
 
 
-#### 3.2.2 Scroll through command history
+#### 3.2.2 Scrolling through command history
 
-OneShelf commands are traversable much like Window's command prompt with the arrow up key traversing into previous commands and arrow down key traversing into next commands.
+OneShelf commands are traversable (up to 20 commands) much like Window's command prompt with the arrow up key traversing into previous commands and arrow down key traversing into next commands.
```

Comment:
I think can be better rephrased with the action first. By pressing arrow up key ........ you can ........ What do you think?

Reply:
Paraphrased! Take a look!